templates/options.erb

```diff
@@ -0,0 +1,15 @@
+Dpkg::Options {
+<%- @_options.each_pair do |config, value| 
+  if %w(force_confdef force_confold force_confnew force_confmiss).include?(config) then
+    if value.is_a?(TrueClass) || value.is_a?(FalseClass) then 
+      if value then -%>
```

Comment:
Why is there an additional check here? We've already checked if they're a boolean or not so they contain a value.